conf/test_panelprep.config

```diff
@@ -21,7 +21,7 @@ params {
 
     // Input data
     input              = "tests/csv/panel.csv"
-    input_region_file  = "tests/csv/regionsheet.csv"
+    input_region       = "tests/csv/regionsheet.csv"
     outdir             = "results/test_panelprep"
```

Comment:
```suggestion
```

outdir should be left empty and specified on the CLI, otherwise it can get messy for people using cloud

Reply:
Okay thanks